pkg/server/wire.go

```diff
@@ -234,6 +234,7 @@ var wireBasicSet = wire.NewSet(
 	datasourceproxy.ProvideService,
 	search.ProvideService,
 	searchV2.ProvideService,
+	searchV2.ProvideSearchHTTPService,
```

Comment:
Rather than a new service... what about adding a function like:
https://github.com/grafana/grafana/blob/main/pkg/services/store/service.go#L65

to the existing one?  with store we had two for a while, but that felt silly after a while

Reply:
I personally like having different files/services for the "access points" (grpc, graphql, rest api, cli, scheduler, mq, ...) and the actual services (business logic), as it helps with keeping the actual services clean of transport-level concepts. 

It can feel silly in smaller services, but I think its a worthy convention in general

Reply:
ok 👍